getting-started/photometry.rst

```diff
@@ -22,107 +20,87 @@ This is the measurement strategy:
 4. :ref:`Merge the multi-band catalogs of source measurements to identify the best positional measurements for each source <getting-started-tutorial-merge-coadds>`.
 5. :ref:`Re-measure the coadds in each band using fixed positions (forced photometry) <getting-started-tutorial-forced-coadds>`.
 
-.. tip::
-
-   Instead of running multiple command-line tasks, like you'll do here, you could instead run the :command:`multiBandDriver.py` command as an integrated multi-band source measurement pipeline.
-
 Set up
 ======
 
-Pick up your shell session where you left off in :doc:`part 4 <coaddition>`.
-That means your current working directory must *contain* the :file:`DATA` directory (the Butler repository).
+Pick up your shell session where you left off in :doc:`part 5 <coaddition>`.
+For convenience, start in the top directory of the example git repository.
+
+.. code-block:: bash
+
+   cd $GEN3_DC2_SUBSET_DIR
 
 The ``lsst_distrib`` package also needs to be set up in your shell environment.
 See :doc:`/install/setup` for details on doing this.
 
 .. _getting-started-tutorial-detect-coadds:
 
-Detecting sources in coadded images
-===================================
+Run detection pipeline task
+===========================
 
-To start, you can detect sources in the coadded images to take advantage of their depth and high signal-to-noise ratio.
-Use the :command:`detectCoaddSources.py` command-line task to accomplish this:
+Processing will be done in two blocks with two different pipelines.
+The first will do steps 1 through 4 in the introduction.
+The end result will be calibrated coadd object measurements and calibrated coadd exposures.
+More high level details are available in the sections that follow.
 
 .. code-block:: bash
 
-   detectCoaddSources.py DATA --rerun coadd:coaddPhot \
-       --id filter=HSC-R tract=0 patch=0,0^0,1^0,2^1,0^1,1^1,2^2,0^2,1^2,2
+   pipetask run -b SMALL_HSC/butler.yaml -d "tract = 9813 AND skymap = 'hsc_rings_v1' AND patch in (38, 39, 40, 41)" -p 'pipelines/DRP.yaml#coadd_measurement' -i u/$USER/coadds --register-dataset-types -o u/$USER/coadd_meas
 
-Notice that since this task operates on coadds, we need to select the coadds using the ``filter``, ``tract``, and ``patch`` data ID keys.
+Notice that since this task operates on coadds, we can select the coadds using the ``tract``, and ``patch`` data ID keys.
+In past sections, the examples left off the ``-d`` argument in order to process all available data.
+This example, however, is selecting just four of the patches for this step.
+Those four patches have coverage from all 40 visits in the tutorial repository which means there doesn't need to be as much fine tuning to configurations, and we can process these patches just as the large scale HSC processing is done.
```

Comment:
I think a comment about why we might need to change configs is warranted here. Even if it is just one line to say config options can be found here and you might want to change some depending on your needs.